reana_client/schemas/reana_analysis_schema.json

```diff
@@ -116,23 +116,29 @@
         }
       }
     },
-    "workflow": {
-      "id": "/properties/workflow",
-      "type": "object",
-      "title": "Analysis workflow.",
-      "description": "Workflow which represents the steps that need to be run to reproduce an analysis.",
-      "properties": {
-        "file": {
-          "id": "/properties/workflow/properties/file",
-          "type": "string",
-          "title": "Workflow file name.",
-          "default": "workflow.yml"
-        },
-        "type": {
-          "id": "/properties/workflow/properties/type",
-          "type": "string",
-          "title": "Workflow engine.",
-          "description": "Name which represents a supported workflow engine to be used to reproduce the analysis."
+    "workflows": {
+      "id": "/properties/workflows",
+      "type": "array",
+      "title": "Analysis workflows.",
+      "description": "Set of workflow which represent the steps that need to be run to reproduce an analysis.",
+      "items": {
+        "id": "/properties/workflows/items",
+        "type": "object",
+        "title": "Analysis workflow.",
+        "description": "Workflow which represent the steps that need to be run to reproduce an analysis.",
+        "properties": {
+          "file": {
+            "id": "/properties/workflows/items/properties/file",
+            "type": "string",
+            "title": "Workflow file name.",
```

Comment:
It is fine as is, but for some reason it got me thinking should it be changed to something like:
"Name of workflow definition file."?
"Workflow definition file name"?
"Workflow definition/specification filename"?

Just did a quick search on reanahub-organization and it seems that 'file name' is used more than 'filename'.

Reply:
Workflow specification filename. Thanks! 👍